src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js

```diff
@@ -185,32 +185,31 @@ export default React.createClass({
             title = _t("Error");
             content = _t("Unable to load backup status");
         } else if (this.state.restoreError) {
-            title = _t("Error");
-            content = _t("Unable to restore backup");
+            if (this.state.restoreError.errcode === MatrixClientPeg.get().RESTORE_BACKUP_ERROR_BAD_KEY) {
```

Comment:
Can we import `MatrixClient` from the js-sdk and use that instead? Accessing static properties on an instance variable is not fun :(